pkg/apis/networking/v1/defaults.go

```diff
@@ -22,6 +22,10 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+var (
+	defaultPathType = networkingv1.PathTypeImplementationSpecific
```

Comment:
inline this...

Reply:
Same conversation here: https://github.com/kubernetes/kubernetes/pull/88041#discussion_r378601492